config/locales/en.yml

```diff
@@ -658,69 +658,81 @@ en:
+
+      <hr class="spacer" />
 
       <h3 id="data-retention">What is your data retention policy?</h3>
 
       <p>We will make a good faith effort to:</p>
 
       <ul>
-        <li>Retain server logs containing the IP address of all requests to this server no more than 90 days.</li>
-        <li>Retain the IP addresses associated with registered users and their posts no more than 5 years.</li>
+        <li>Retain server logs containing the IP address of all requests to this server, in so far as such logs are kept, no more than 90 days.</li>
+        <li>Retain the IP addresses associated with registered users no more than 5 years.</li>
       </ul>
 
+      <p>You may irreversibly delete your account at any time.</p>
+
+      <hr class="spacer"/>
+
       <h3 id="cookies">Do we use cookies?</h3>
 
       <p>Yes. Cookies are small files that a site or its service provider transfers to your computer's hard drive through your Web browser (if you allow). These cookies enable the site to recognize your browser and, if you have a registered account, associate it with your registered account.</p>
 
-      <p>We use cookies to understand and save your preferences for future visits and compile aggregate data about site traffic and site interaction so that we can offer better site experiences and tools in the future. We may contract with third-party service providers to assist us in better understanding our site visitors. These service providers are not permitted to use the information collected on our behalf except to help us conduct and improve our business.</p>
+      <p>We use cookies to understand and save your preferences for future visits.</p>
+
+      <hr class="spacer" />
 
       <h3 id="disclose">Do we disclose any information to outside parties?</h3>
 
-      <p>We do not sell, trade, or otherwise transfer to outside parties your personally identifiable information. This does not include trusted third parties who assist us in operating our site, conducting our business, or servicing you, so long as those parties agree to keep this information confidential. We may also release your information when we believe release is appropriate to comply with the law, enforce our site policies, or protect ours or others rights, property, or safety. However, non-personally identifiable visitor information may be provided to other parties for marketing, advertising, or other uses.</p>
+      <p>We do not sell, trade, or otherwise transfer to outside parties your personally identifiable information. This does not include trusted third parties who assist us in operating our site, conducting our business, or servicing you, so long as those parties agree to keep this information confidential. We may also release your information when we believe release is appropriate to comply with the law, enforce our site policies, or protect ours or others rights, property, or safety.</p>
 
```

Comment:
Any trusted third parties should be explicitly listed, so that users can seek the corresponding privacy policies and understand who is accessing their data.

Reply:
"agree to keep the information confidential" is not a particularly strong protection, as there are many ways to misuse data that don't involve sharing it with others.   
Ideally, no no information would be transferred to third parties unless they were also:
bound to not use the information for any purposes outside of assisting us in operating the site, conducting our business and providing the service.  
Also, these guarantees should apply not just to PII, but to any data the user has not made public.

Reply:
We need to distribute such a privacy policy that would be valid for all installations. So it can't get too specific. I doubt that admins will go hunting down the, say, DigitalOcean privacy policy, or Mailgun privacy policy (please mind that admins *can* completely rewrite the privacy policy themselves from the admin interface, if they decide to include that information)

You also have to see this in perspective to the prior privacy policy. Is it overall an improvement, or is it not? I can't fix everything at once.

Reply:
Without an enforcement provision, the policy is fairly meaningless.  You make it clear that the policy is merely a "good faith" effort in which the weakest link is the default for privacy.  What is that actual mechanism for safeguarding privacy from abuse?  I'll be honest and suggest that as a new user, I took a dim view of the privacy options offered and assume they are no better than any other service but that there was less crass commercialization of my own posts.  It's hard to imagine that even the metadata provided is necessary for the improvement of the user experience as users tend to provide concrete feedback on services and will let you know what they want changed.  As for "private communications", the service appears to provide no real protection at all.  Unless users were to communicate with some kind of trusted encryption, it appears that private communications are somewhat open.  The ACLU has a 45 page report on "Informational Privacy in the Digital Age" which might provide you with some background as to the complexity of the issues at hand- but no one read reports.